Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-04-25
Initial Package Version: 4.2.0
Origin: Fedora Project 
        http://cvs.fedora.redhat.com/viewcvs/*checkout*/rpms/ntp/devel/
Description: Fix compilation issues with gcc4

--- ntp-stable-4.2.0a-20040617/include/ntp_stdlib.h.gcc4	2005-03-08 14:29:42.940559784 +0100
+++ ntp-stable-4.2.0a-20040617/include/ntp_stdlib.h	2005-03-08 14:29:43.015548384 +0100
@@ -133,10 +133,6 @@
 extern u_char *	cache_key;		/* key pointer */
 extern u_int	cache_keylen;		/* key length */
 
-/* clocktypes.c */
-struct clktype;
-extern struct clktype clktypes[];
-
 /* getopt.c */
 extern char *	ntp_optarg;		/* global argument pointer */
 extern int	ntp_optind;		/* global argv index */
--- ntp-stable-4.2.0a-20040617/include/ntp_refclock.h.gcc4	2004-02-25 06:57:56.000000000 +0100
+++ ntp-stable-4.2.0a-20040617/include/ntp_refclock.h	2005-03-08 14:29:43.015548384 +0100
@@ -65,6 +65,7 @@
 	const char *clocktype;	/* long description */
 	const char *abbrev;	/* short description */
 };
+extern struct clktype clktypes[];
 
 /*
  * Configuration flag values
--- ntp-stable-4.2.0a-20040617/include/ntpd.h.gcc4	2004-05-25 13:02:24.000000000 +0200
+++ ntp-stable-4.2.0a-20040617/include/ntpd.h	2005-03-08 14:29:43.016548232 +0100
@@ -225,8 +225,6 @@
 #endif
 
 /* ntp_control.c */
-struct ctl_trap;
-extern struct ctl_trap ctl_trap[];
 extern int	num_ctl_traps;
 extern keyid_t	ctl_auth_keyid;		/* keyid used for authenticating write requests */
 
--- ntp-stable-4.2.0a-20040617/include/ntp_control.h.gcc4	2004-05-25 13:02:23.000000000 +0200
+++ ntp-stable-4.2.0a-20040617/include/ntp_control.h	2005-03-08 14:29:43.016548232 +0100
@@ -268,6 +268,7 @@
 	u_char tr_flags;		/* trap flags */
 	u_char tr_version;		/* version number of trapper */
 };
+extern struct ctl_trap ctl_trap[];
 
 /*
  * Flag bits
